tests/artifacts/test_artifacts.py

```diff
@@ -221,6 +268,9 @@ def test_log_artifact(
     expected_target_path: str,
     artifact_path: str,
     generate_target_path: bool,
+    tag: str,
+    raises: typing.Union[ABCMeta, ValueError],
```

Comment:
Couple of things:
1. The typing union if you're already using it should be `typing.Union[ABCMeta, _pytest.python_api.RaisesContext]
As:
```
In [4]: type(pytest.raises(mlrun.errors.MLRunInvalidArgumentError))                                                                                                                                                                     
Out[4]: _pytest.python_api.RaisesContext
```
Because of this I personally would just set a `typing.Any` here.
2. The name `raises` is ok, But I personally prefer `expectation` as that describes it's value better: the expectation is either raising MLRunInvalidArgumentError or the expectation is `does_not_raise`